### PR TITLE
Postgres integration

### PR DESCRIPTION
**Pull request by kouridis**

### CODE REVIEW

File: macros/edr/data_monitoring/monitors_query/dimension_monitoring_query.sql [hunk 1]
```diff
@@ -46,7 +46,7 @@
                     dimension_value,
                     1 as joiner
                 from filtered_monitored_table
-            )
+            ) rs
```
Comment (elongl) [inline macros/edr/data_monitoring/monitors_query/dimension_monitoring_query.sql:49]: What are those?

Comment (kouridis) [reply macros/edr/data_monitoring/monitors_query/dimension_monitoring_query.sql:49]: In postgres subqueries in FROM must have an alias

Comment (elongl) [reply macros/edr/data_monitoring/monitors_query/dimension_monitoring_query.sql:49]: I see. What does `rs` stand for?

Comment (kouridis) [reply macros/edr/data_monitoring/monitors_query/dimension_monitoring_query.sql:49]: result set. I use it in my queries

**Comment by elongl** [comment]
Merging this so we can continue the work on this branch!

**Pull request merged**